src/main/java/duke/commands/AddCommand.java

```diff
@@ -0,0 +1,94 @@
+package duke.commands;
```

Comment:
Good use of OOP principles by abstracting out all the commands into separate classes

Reply:
Thanks!